app/src/main/java/com/codesoom/assignment/Exception/TaskNotFoundException.java

```diff
@@ -0,0 +1,25 @@
+package com.codesoom.assignment.Exception;
+
+/**
+ * TaskNotFoundException은 taskManger에서 존재하지 않는 Task에 접근했을 때, 발생하는 Exception 입니다.
+ *
+ * @author wjy5446
+ */
+
+public class TaskNotFoundException extends RuntimeException {
```

Comment:
예외가 발생한다는 것은 내부 구조를 모르는 사용자의 입장입니다. 예외를 작성할 때는 코딩하는 사람의 입장에서 생각해 보세요.

```suggestion
/**
 * 할 일을 찾지 못했을 때 던집니다.
 */
public class TaskNotFoundException extends RuntimeException {
```

다음은 흔하게 볼 수 있는 `NullPointerException`의 javadoc입니다.

![image](https://user-images.githubusercontent.com/1855714/130070849-695ae8eb-e1ca-4730-9b78-b95b39014076.png)

summary 부분을 잘 읽어 봅시다. 한국어로 번역은 제가 했스니다.

> "Thrown when an application attempts to use null in a case where an object is required."
>
> "애플리케이션이 객체를 필요로할 때 null을 사용하게 되면 던집니다."

다음은 먼저 사용하셨던 `NoSuchElementException`의 javadoc입니다. 이것도 읽어 봅시다.

![image](https://user-images.githubusercontent.com/1855714/130070728-353849ec-1eca-4efd-984d-2b1c3eef0808.png)

다른 것도도 읽어 봅시다.

![image](https://user-images.githubusercontent.com/1855714/130071341-f4c6c9c4-e1d4-412f-bd6b-93dd8abe0f89.png)

javadoc을 어떻게 작성해야 할 지 모르겠다면 우리가 늘 사용하고 있는 클래스 (`String`, `Integer`, `NullPointerException`, ...) 들을 열고 주석을 읽어 보는 것이 가장 좋은 학습법입니다.

Reply:
어떻게 예외를 작성할 지 모르고 있었는데, 좋은 팁 알려주셔서 감사합니다.